mkosi.extra/usr/lib/tmpfiles.d/etc.conf

```diff
@@ -9,14 +9,17 @@ L /etc/locale.conf
 L /etc/nsswitch.conf
 L /etc/issue
 L /etc/profile
+# Required by authselect
+L /etc/authselect
```

Comment:
I think it would make sense for this to go into a `mkosi.extra` specific for Fedora, since other distributions don't use it and this will create a dangling symlink then.

Reply:
ack, updated